torchvision/transforms/functional.py

```diff
@@ -84,8 +84,7 @@ def to_tensor(pic):
 
     img = img.view(pic.size[1], pic.size[0], len(pic.getbands()))
     # put it from HWC to CHW format
-    # yikes, this transpose takes 80% of the loading time/CPU
```

Comment:
I think this comment might still be valid (as the transpose is not the culprit, but the contiguous call which used to be slow. But `contiguous` has been optimized quite a bit since then, so might not be a problem anymore